packages/server/src/login/http-signature.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import * as fs from "fs";
+import * as httpSignature from 'http-signature';
+import { Request } from 'express';
+
+
+@Injectable()
+export class HttpSignatureService {
+    public verifyRequest(req: Request): boolean {
+        const PUBLIC_KEY = fs.readFileSync(__dirname + '/khoury-public-key.pem', 'ascii');
```

Comment:
Not sure if this could be made a class variable so for better performance?

Reply:
Yes please do this in the constructor

Reply:
done